options.go

```diff
@@ -148,7 +148,7 @@ func DefaultOptions(path string) Options {
 		NumCompactors:           4, // Run at least 2 compactors. Zero-th compactor prioritizes L0.
 		NumLevelZeroTables:      5,
 		NumLevelZeroTablesStall: 15,
-		NumMemtables:            5,
+		NumMemtables:            15,
```

Comment:
performance/memory implications?

Reply:
switched it back to 5 keeping the read speed in mind.